tests/integration/test_alert_rules_fire.py

```diff
@@ -70,6 +69,8 @@ async def test_alert_rules_do_forward_to_alertmanager(ops_test, loki_charm, loki
     alertmanager_app_name = "alertmanager"
     app_names = [loki_app_name, tester_app_name, alertmanager_app_name]
 
+    await ops_test.model.set_config({"logging-config": "<root>=WARNING; unit=DEBUG"})
+
```

Comment:
Looks like a debug statement that should be removed prior to merging?

Reply:
I am happy to remove. Pinging @sed-i to get a second opinion if this is still required and why it was added in the first place.

Reply:
It's not needed. It was added so one of the models hanging (either at `0/1` units or `Installing agent`) would have more rigorous logs "out of the box".